src/services/analytics/index.ts

```diff
@@ -0,0 +1,15 @@
+import { AnalyticsEvent } from './types'
+import { IS_PRODUCTION } from '@/config/constants'
+import { gtmTrack } from './gtm'
+
+export const trackEvent = (eventData: AnalyticsEvent) => {
+  if (!IS_PRODUCTION) {
```

Comment:
I think this is the cause of @francovenica's issues regarding no logging.

Reply:
Let's enable logging on prod for now. 👍 I'll remove the check.